virttest/guest_agent.py

```diff
@@ -15,15 +15,10 @@
     logging.warning("Could not import json module. "
                     "virt agent functionality disabled.")
 
+from .qemu_monitor import Monitor, MonitorError
+from . import error_context
```

Comment:
Any reason to update this two?

Reply:
Actually I didn't update these, it's created when I do push operation.The same with L537

Reply:
@xiagao Well, I guess you need to rebase your branch.

Reply:
@vivianQizhu Thanks. Rebase is done and test result is pass.
 (1/2) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_guest_file.q35: PASS (42.22 s)
 (2/2) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_guest_file.q35: PASS (85.41 s)